pyvelox/pyvelox.cpp

```diff
@@ -28,9 +28,19 @@ std::string serializeType(const std::shared_ptr<const velox::Type>& type) {
 #ifdef CREATE_PYVELOX_MODULE
 PYBIND11_MODULE(pyvelox, m) {
   m.doc() = R"pbdoc(
-        PyVelox native code module
-        -----------------------
-       )pbdoc";
+      PyVelox native code module
+      --------------------------
+
+      .. currentmodule:: pyvelox.pyvelox
+
+      .. autosummary::
+         :toctree: _generate
+         
+         ARRAY
```

Comment:
What does this code do? Why it lists ARRAY and ROW, but not MAP?

Reply:
Basically, we have to manually write down the exposed endpoints in the `pyvelox.pyvelox` package. 
This was the approach I found to generate the Python docs from the Pybind11->Sphinx. I merely added this as an example to show how to add docs. I think there will be a follow up PR to cover the complete documentation guide. 

cc @kgpai

Reply:
Would it make sense to add a comment about this? BTW, where can I see the documentation generated by this PR?

Reply:
Yes definitely. I will add some comments.

Reply:
Lets remove ARRAY-ROW etc and just have the outline if we plan to rewrite it in a succeeding PR.